src/core/Akka.Cluster/Cluster.cs

```diff
@@ -62,13 +62,29 @@ public static Cluster Get(ActorSystem system)
             return system.WithExtension<Cluster, ClusterExtension>();
         }
 
+        static Cluster()
+        {
+            bool GetAssertInvariants()
+            {
+                var isOn = Environment.GetEnvironmentVariable("akka.cluster.assert")?.ToLowerInvariant();
```

Comment:
assert invariants only runs when the `akka.cluster.assert` environment variable is set to `on`